src/utilities/spacing.scss

```diff
@@ -0,0 +1,38 @@
+@import "./defaults.scss";
+
+// --- Margin + Padding Spacing Classes ---
+
+@each $spaceClass, $space in $spacing {
+  // classes for all sides
+  .m-#{$spaceClass} {
+    margin: $space;
+  }
+
+  .p-#{$spaceClass} {
+    margin: $space;
```

Comment:
`.p-{size}` classes are setting margin instead of padding

Reply:
@ethanWallace good catch, updated!